.github/validation_grid_utils.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env -S uv run -q --script
+
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+# "typer>=0.10",
+# "icon4py-testing",
+# ]
+# [tool.uv.sources]
+# icon4py-testing = {path = "../model/testing"}
+# ///
+
+
+import typer
+import sys
+
+from icon4py.model.testing import grid_utils
+from icon4py.model.testing import definitions
+import hashlib
+
+VALIDATION_GRIDS = (definitions.Grids.R02B04_GLOBAL, definitions.Grids.MCH_CH_R04B09_DSL, definitions.Grids.MCH_OPR_R04B07_DOMAIN01) # change to MCH_OPR_R04B07_DOMAIN01
+app = typer.Typer()
+@app.command(name="cache-key")
```

Comment:
It seems that pre-commit / ruff-format is not processing this file. Do you know why? Could you add it?

Reply:
I guess it is this in the `.pre-commit-config.yaml`:
```
minimum_pre_commit_version: 3.8.0
files: |
  (?x)^(
      model/.*|
      tools/.*
  )$

```